upb/upb.c

```diff
@@ -67,6 +67,18 @@ static void *upb_global_allocfunc(upb_alloc *alloc, void *ptr, size_t oldsize,
   }
 }
 
+static uint32_t* upb_cleanup_pointer(void* cleanup_metadata) {
+  return (uint32_t*)((uintptr_t)cleanup_metadata & ~0x1);
+}
+
+static bool upb_cleanup_has_initial_arena(void* cleanup_metadata) {
```

Comment:
`has_initial_block` instead of `has_initial_arena`?

Reply:
Done